### ISSUE TITLE
Missing TV show genre images

### ISSUE THREAD

**Issue by juanma-cvega** [jurialmunkey/plugin.video.themoviedb.helper#255]
This is more of a skin issue but it's related to TheMovieDB Helper. Is there any TV show genre pack that can be installed that has the missing icons for TV shows like Action & adventure?

**Comment by drinfernoo** [comment]
That's not really related to the skin *or* the helper... as it would be a completely separate resource add-on.

**Comment by juanma-cvega** [comment]
Yes, it has nothing to do with neither of them but given it is configured from the skin and TheMovieDB Helper is the one that makes use of it, I was wondering if this has come up before and if someone knows of a genre pack that includes the missing icons. Maybe it could be added, if it exists, as a dependency to the skin or this addon that could be installed alongside them. Or at least added to the list of dependencies of the skin that the user could decide to install.

**Comment by jurialmunkey** [comment]
None that I know of. However, You can use TextureTool to unpack the Textures.xbt in the resource addon and then add a png that matches the missing genre name and re-pack Textures.xbt

https://kodi.wiki/view/TextureTool
